### PR TITLE
options: add set_verify_checksums and set_level_compaction_dynamic_level_bytes

### PR DESCRIPTION
**Pull request by ordian**
Part of #260.

### CODE REVIEW

File: src/db.rs [hunk 1]
```diff
@@ -1930,6 +1930,12 @@ impl ReadOptions {
         unsafe { ffi::rocksdb_readoptions_set_total_order_seek(self.inner, v as c_uchar) }
     }
 
+    pub fn set_verify_checksums(&mut self, v: bool) {
```
Comment (aleksuss) [inline src/db.rs]: Add comment docs for method please.

**Comment by aleksuss** [comment]
It would be cool if you'll add a unit tests for these methods.

**Comment by ordian** [comment]
@aleksuss please take another look

**Comment by aleksuss** [comment]
One more thing. `cargo fmt` apply please.

**Pull request merged**